src/main/java/employment/system/controllers/ApplyForJobController.java

```diff
@@ -0,0 +1,99 @@
+package employment.system.controllers;
+
+
+import employment.system.checkers.EmailValidation;
```

Comment:
Change EmailValidation with EmailChecker.

Reply:
I will do that in the next story ( ESFI-27 - Refactoring) because we decided to modify some issues there.